.travis.yml

```diff
@@ -35,7 +35,7 @@ cache:
 services:
   - docker
 script:
-  - docker create -v ${TRAVIS_BUILD_DIR}:/travis --name travis debian
-  - docker run --env ARCH=${ARCH} --env LD=${LD} --rm --volumes-from travis debian /bin/bash -c 'cd /travis && ./debian-setup.sh && ./driver.sh && ccache -s'
+  - docker create -v ${TRAVIS_BUILD_DIR}:/travis --name travis debian:sid
+  - docker run --env ARCH=${ARCH} --env LD=${LD} --rm --volumes-from travis debian:sid /bin/bash -c 'cd /travis && ./debian-setup.sh && ./driver.sh && ccache -s'
```

Comment:
What's the default if you don't specify?

Reply:
Stable, I think:
```
$ docker run -it debian cat /etc/os-release
PRETTY_NAME="Debian GNU/Linux 9 (stretch)"
NAME="Debian GNU/Linux"
VERSION_ID="9"
VERSION="9 (stretch)"
```

Reply:
Yeah, it's just a Stretch image by default.